### DISCUSSION TITLE
Item Labels for iFrames

### DISCUSSION THREAD

**Discussion by renecnielsen** [knowledgefutures/pubpub#2422] [Ideas]
I currently use iFrames for Flourish and Datawrapper charts, and would really like the opportunity to reference those and number them along with static image charts.

Would it be possible to basically add iFrame (blocks?) to the Item Labels configuration? That is, I could set images and iFrames to be "Figures" and reference iFrames in the same way I can reference images?

And just to be sure, should oEmbed support be added (#2420), I would want those to be included as well ;)

**Comment by tosteiner** [comment]
definitely a +1 from me 👍🏾
